### ISSUE TITLE
TRACK: Update on new issues 28.10.23

### ISSUE THREAD

**Issue by cmarooney-stfc** [pace-neutrons/Horace#1361]
(a) 
Update, tracking new issues since previous tracking issues were created.
Not yet prioritized apart from labels
(b)
Summary of remaining open issues (not low priority, future or post-construction) recorded in previous TRACK issues by theme - to enable continued prioritization.

THIS MEETING 2.11.23 sprint review

Jacob to make ticket for documentation of Alex fileabcked apply
Emphasize differences in various types of filebacked 
matlab help pageop base is existing doc within codebase as done by ALex on which we can build
identifies planning - what are the key horace points where we need to be aware of speed - meeting Toby Jacob, followed by meetings with excitations people to get data. Jacob to organize meeting with Toby (zoom)

----

(a) New issues since issue 1272, for prioritization. Grouped by theme

Validation
- [ ] Alex #1314 - estimate start for this is 20th Nov - implies not look at resconv ticket #1348 beforehand

Bugs
 - [x] Alex #1360 [BUG, DISCUSSION] LOW priority as it relates to focussed projection not being asked for at the moment
 - [x] Alex #1359 [BUG, EPIC, duplicated in Refactor] split epic to separate 2 types of op, put relevant tickets for energy focus together; LOW priority for all
 - [ ] Duc #1355 [BUG, also parallel] Alex thinks this is fixed PR #1365, waiting Duc review 
 - [ ] Alex #1350 [BUG, also Detectors] HIGH priority for Chris : debugging paused as not reproducible, see comments in ticket.
 - [ ] Alex #1303 [not marked BUG but inferred from code inspection] HIGH in progress with Alex
 - [ ] Alex #1302 [also EPIC for refactor with Apply] HIGH in progress with Alex (the filebacked epic)

Tests
 - [ ] Alex #1345 HIGH priority to make tests (1d) after filebacked, rename to get_mask_for_pix, not related to filebacked

Refactor
 - [x] Alex #1359 [duplicated in BUG, EPIC] commented in bugs above
 - [x] Alex #1356 [refactor to new serializable] DONE awaiting final merge
 - [ ] Alex #1347 [also Filebacked] HIGH in progress filebacked epic
 - [ ] Alex #1344 [also Filebacked] HIGH ditto but review functionality first - before next sprint when Toby returns, Toby sets up meeting
 - [ ] Alex #1302 [also BUG] see above Bugs
 
Parallel
 - [ ] Duc #1355 [BUG, also parallel] see above
 - [ ] Aaron #1289 [BUG not labelled] part of Alex review #1314

Performance
 - [ ] Alex #1354 [nolabel] see ABove Alex epic to be split

Mex/C++
 - [x] Alex #1295 LOW priority (optimization) - awaits benchmarking results

Filebacked
 - [ ] Alex #1352 HIGH part of Alex peic
 - [ ] Alex #1347 [also Refactor]
 - [ ] Alex #1344 [also Refactor]
 - [ ] Alex #1341 Jacob to do as part of docs - issues arising may involve code fixes
 - [ ] Alex #1330 in progress part of epic
 - [ ] Alex #1320 ditto
 - [x] Alex #1319 LOW priority
 - [ ] Alex #1303 [not marked BUG but inferred from code inspection] see above

Tobyfit
 - [ ] Alex #1348 - transfer to Jacob for minimal update
 - [ ] Chris #1299 [also Documentation]
 - [ ] Chris #1298 HIGH

 Detectors
 - [ ] Alex #1350 [also BUG]
 - [ ] Chris #1340 HIGH

Serialize
 - [ ] Alex #1328 [knockon from BUG #1324] Jacob to follow up HIGH
 
Documentation
 - [ ] Chris #1299 [also Tobyfit, needs Doc label]

Discussions (?low priority before end construction)
 - [ ] Alex #1358 [DISCUSSION] LOW but may be able to resolve quickly
 - [ ] Chris #1338 
 - [ ] Chris #1318 [? add DISCUSSION label?] high priority LOW urgency fold in with algorithm docs

Unlabelled
 - [ ] Alex #1354 [nolabel]

(b) OLDER ITEMS FROM LAST PRIORITISATION - to be reassessed against later tickets above

Bugs
#1025 referenced in #1268 check if done (Chris)
#935 may be fixed by #801 check (Toby)

Instruments
#33 - Chris - part of Duc nxspe input
also part of #21 remaining detector stuff for SQW needed to finish this epic
#1266 detector issues, various, for Chris and Toby

Documentation
#451 assigned Jacob

Mex
#1269 tracking ticket - 3x issues unprioritized

Performance
#901 in tracking set #1265 medium priority, also Tobyfit

Masking
#922 for Toby IX_mask etc refactor

Projections
#1195 still high priority
#954  alex report on spherical projections

CI
#1054 for Jacob
Aaron may don #676 and 598

Refactoring
#1142 with Alex MEDIUM
#1241 with Toby
#1107 Alex is to determine priority of this one
#1062 Jacob to lead discussion (a) is this post-construction (b) does it duplicate #1243

Serializable
#1147 MEDIUM
#1061 Toby to set priority
#1051 Jacob to check if done
#905 MEDIUM
#1146 may have been done need to find PR ***

Testing
#1080 should Toby have closed this?
#1052 Chris HIGH
#962 "DISCUSSION - SEE OTHER REFERENCES" - ckeck this
#937 check MEDIUM priority with Toby
#848 Chris PR open #1284

Containers
#1157 Chris HIGH
#1078 Chris HIGH

Documentation
#1159 Chris PR #1285
#1199  Jacob getting Russell images
#616 Jacob, ongoing
#579 ??? think this should be LOW but nothing recorded
#1274 developer docs - for us to discuss parameters

Parallel
#1173 waiting on Alex apply refector
#1172 low priority as Jenkins
#1203 Jacob HIGH
#488 part of Aaron work

Filebacked
#1214 Alex retesting
#798 Jacob MEDIUM

Plotting
#1185 Jacob (a) does it duplicate #808? (b) is it easy to fix
#1042 Alex to see what is left
#808 See #1185 above

i/o
#1186 FILEBACKED PRIORITY
#204 parked for review. As this is SPE readers and we may need them for instrument data, possibly HIGH

**Comment by cmarooney-stfc** [comment]
@tgperring @abuts @oerc0122 I have updated our tracked priority issues with more recent tickets. For your convenience, should you wish to look at this list for Tuesday's sprint review. I will be back on Wednesday. Thanks, Chris
